src/mpc/faand.rs

```diff
@@ -922,7 +1109,26 @@ type Bucket<'a> = Vec<(&'a Share, &'a Share, &'a Share)>;
 /// Protocol Pi_aAND that performs F_aAND from the paper
 /// [Global-Scale Secure Multiparty Computation](https://dl.acm.org/doi/pdf/10.1145/3133956.3133979).
 ///
-/// The protocol combines leaky authenticated bits into non-leaky authenticated bits.
+/// This protocol converts *leaky authenticated AND triples* into
```

Comment:
This might be a bit misleading, as the input `xyr_shares` are not leaky authenticated triples, but just random authenticated shares from `fashare`. The leaky triples are only internally created by `flaand` from these `xyr_shares`.